packages/shared/src/components/notifications/EmailNotificationsTab.tsx

```diff
@@ -132,6 +132,24 @@ const EmailNotificationsTab = (): ReactElement => {
               )
             }
           />
+          <NotificationSwitch
+            id="opportunities"
+            label="Personalized job opportunities"
+            description={
+              <>
+                Get notified only when there&apos;s a role that fits your skills
+                and preferences. No spam, no pressure.
+              </>
+            }
```

Comment:
Why not just a string?

```suggestion
            description="Get notified only when there's a role that fits your skills and preferences. No spam, no pressure."
```